src/app/utils/common.util.ts

```diff
@@ -33,18 +33,18 @@ export class CommonUtil {
     let unit = 'MB';
     let toValue = +value;
 
-    if (toValue / 1024 >= 0.9) {
-      toValue = toValue / 1024;
+    if (toValue / 1000 >= 1) {
+      toValue = toValue / 1000;
```

Comment:
We need to be careful here: what do we get from the system? Is that MB or is it really MiB?
It goes all the way back to the shim, the core does not care and uses whatever it gets as the value without scales. If I read the shim code correctly it all gets converted to MB for memory and rounded into whole MB's (1000 * 1000 bytes).
Please confirm that I am reading the shim code correctly

Reply:
@wilfred-s  @yangwwei  Could you please confirm that we are getting memory in MB (not in MiB) by referencing the shim code?

Reply:
Here is the code to parse the resources: https://github.com/apache/incubator-yunikorn-k8shim/blob/03f36354820033a22556f1c82d7be8ef815a99f4/pkg/common/resource.go#L89-L118. We are calling the K8s utility `resource.ParseQuantity()`, so it supports both MB, and MiB. When sending to the core, the shim uses the `MB` as the convention. So this looks good.

Reply:
thanks for the confirmation, I was reading the code correctly then.